src/app/mesh/qgsmeshlayerproperties.cpp

```diff
@@ -91,6 +93,18 @@ QgsMeshLayerProperties::QgsMeshLayerProperties( QgsMapLayer *lyr, QgsMapCanvas *
   if ( !mMeshLayer->styleManager()->isDefault( mMeshLayer->styleManager()->currentStyle() ) )
     title += QStringLiteral( " (%1)" ).arg( mMeshLayer->styleManager()->currentStyle() );
   restoreOptionsBaseUi( title );
+
+  QPushButton *btnStyle = new QPushButton( tr( "Style" ) );
```

Comment:
cannot be the push button part of the UI file?

Reply:
no, as this button take the place of the reset button on the box button, with it in ui files, it leads to have the text "reset" too and picture.
With this implementation, we don"t need to replace the text and remove the image.